server/middlewares/Authorization.js

```diff
@@ -0,0 +1,72 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+import models from '../models';
+
+config();
+
+const { User, DroppedToken } = models;
+
+/**
+ * Middleware to check user access
+ * @exports UserController
+ * @class Authorization
+ */
+class Authorization {
+  /**
+    * Get token from req
+    * @param  {object} req - Request object
+    * @returns {string} token
+    * @static
+    */
+  static getToken(req) {
+    const bearerToken = req.headers.authorization;
+    const token = bearerToken && bearerToken.replace('Bearer ', '');
```

Comment:
token would be `false` if `bearerToken` does not exist

You should properly change this to something like:

```javascript
if (bearerToken && bearerToken.startsWith('Bearer ')){
  const token = bearerToken.replace('Bearer ', '');
  return token
} else if (bearerToken){
  return bearerToken; // This part is not entirely needed (only gives the option for the token to be set in the header directly
}else {
  throw new Error('Invalid token supplied');
}
```

then move the `const token = Authorization.getToken(req);` into the `try/catch` block